build.sbt

```diff
@@ -90,7 +89,7 @@ lazy val dataplugFacebook = Project(id = "dataplug-facebook", base = file("datap
   .settings(buildSettings)
   .enablePlugins(AshScriptPlugin)
   .settings(packageSettings)
-  .dependsOn(dataplug)
+  .dependsOn(dataplug % "test->test;compile->compile")
```

Comment:
Just a question, what does the new addition mean? What does it do?

Reply:
it means the test classes of `dataplug-facebook` can use (compile wise) the test classes of `dataplug` (`compile` does the same for source code that sits under `app` since they are Play projects)

Reply:
before there was no scoping of dependencies and all the projects had test dependencies deployed... this is fixed

Reply:
Oh, I see!!! Thanks for the info!!!